extensions/http(s)/test/test_helper.exs

```diff
@@ -1,2 +1,16 @@
 {:ok, _} = Application.ensure_all_started(:httpoison)
 ExUnit.start()
+
+defmodule Raxx.Forwarder do
+  use Raxx.Server
```

Comment:
so calling use Raxx.Server adds a behaviour to the module that expects 4 callbacks to be implemented. The warning you see is because you have not implemented a `handle_info/2` callback. to clear the error I would just add a no op case
```elixir
def handle_info(message, state) do
  raise "I can't see any good reason why handle info would have been called in this test"
end
```